devtools/create_project/create_project.cpp

```diff
@@ -1546,20 +1534,38 @@ void ProjectProvider::createProject(BuildSetup &setup) {
 
 		createProjectFile(setup.projectName, svmUUID, setup, setup.srcDir, in, ex);
 	} else if (!setup.devTools) {
-		// Last but not least create the main project file.
 		in.clear(); ex.clear();
-		// File list for the Project file
+		createModuleList(setup.srcDir + "/video", setup.defines, setup.testDirs, in, ex);
+		createProjectFile("Video", addCommonProjectUUID("Video"), setup, setup.srcDir + "/video", in, ex);
```

Comment:
I am not using Windows or MSVC, but it appears to me that you are adding here project file starting with a capital letter, while all other projects are lowercase. Is it intentional?

Reply:
Just sloppy.

Reply:
Great, then there is no issue with fixing those :D